.github/workflows/test-action-build-init.yml

```
@@ -116,6 +116,7 @@ jobs:
           ${{ matrix.skip-init }} || ${{ matrix.provide-link-targets == '' }} || grep -- '^:orphan:$' rst/_targets.rst || exit 1
           ${{ matrix.skip-init }} || ${{ matrix.provide-link-targets == '' }} || grep -- '^.. _outside_reference_1:$' rst/_targets.rst || exit 1
           ${{ matrix.skip-init }} || ${{ matrix.provide-link-targets == '' }} || grep -- '^.. _outside_reference_2:$' rst/_targets.rst || exit 1
+          ${{ matrix.skip-init }} || ${{ matrix.provide-link-targets == '' }} || (pip install rstcheck && rstcheck rst/_targets.rst) || exit 1
```

```suggestion
          ${{ matrix.skip-init }} || ${{ matrix.provide-link-targets == '' }} || (pip install rstcheck && rstcheck --report-level WARNING rst/_targets.rst) || exit 1
```
trying out changing the reporting level:
https://rstcheck.readthedocs.io/en/latest/usage/cli/#cmdoption-rstcheck-report-level

---

Ah, I see. Ignore my chatter :)